tests/collections/llm/test_hf_import.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+from pathlib import Path
+
+from nemo import lightning as nl
+from nemo.collections import llm
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Test Llama2 7B model model conversion from HF')
+    parser.add_argument('--hf_model', type=str, help="Original HF model")
+    parser.add_argument('--output_path', type=str, help="NeMo 2.0 export path")
+
+    return parser.parse_args()
+
+
+if __name__ == '__main__':
+    args = get_args()
+
+    model = llm.LlamaModel(config=llm.Llama2Config7B)
+    nemo2_path = llm.import_ckpt(model, "hf://" + args.hf_model, output_path=Path(args.output_path))
+
+    trainer = nl.Trainer(
```

Comment:
1. In `Trainer`: are `devices` and `tensor_model_parallel_size` correctly set? I think they should depend on `args`.
2. Can we make `MegatronMixedPrecision` to use model precision: BF16 or FP16?

Reply:
Discussed offline: There are issues with distributed fabric import + it's just for creating a nemo2 instance